src/effects/effectparameterslotbase.cpp

```diff
@@ -79,4 +79,13 @@ void EffectParameterSlotBase::clear() {
     m_pControlValue->setDefaultValue(0.0);
     m_pControlType->forceSet(0.0);
     emit(updated());
-}
\ No newline at end of file
+}
+
+void EffectParameterSlotBase::reload() {
+    if (m_pManifestParameter && m_pManifestParameter->showInParameterSlot()) {
+        m_pControlLoaded->forceSet(1.0);
+    } else {
+        m_pControlLoaded->forceSet(0.0);
```

Comment:
This alone breaks controller mappings. Hidden parameters do need to be unloaded from the parameter slot. Merely changing `m_pControlLoaded` does hide parameters from skins, but the ControlObjects are still used by hidden parameters. For example, if the first parameter is hidden, a controller knob mapped to `[EffectRack1_EffectUnitX_EffectY], parameter1` still controls the hidden parameter instead of manipulating the first parameter shown on screen.